### PR TITLE
Add support for 'get-lease-hostnames' option for subnets

### PR DESCRIPTION
**Pull request by ProFfeSsoRr**

### CODE REVIEW

**Comment by k0ste** [comment]
> The _get-lease-hostnames_ statement is used to tell dhcpd whether or not to look up the domain name corresponding to the IP address of each address in the lease pool and use that address for the DHCP _hostname_ option.  If _flag_ is **true**, then this lookup is done for all addresses in the current scope.  By default, or if _flag_ is **false**, no lookups are done.

So, the value should be bool, not on/off. Also this statement must be available not only for subnet but for host, group, pool.

**Comment by ProFfeSsoRr** [comment]
Fixed

**Review by k0ste — CHANGES_REQUESTED** [review]

**Comment by ProFfeSsoRr** [comment]
Fix

File: templates/dhcpd_pool.j2 [hunk 1]
```diff
@@ -12,5 +12,8 @@
 {% if p.filename is defined and p.filename != '' %}
 {{ '    ' + 'filename \"' + p.filename + '\";' }}
 {% endif %}
+{% if p.get_lease_hostnames is defined and p.get_lease_hostnames in ['true', 'false'] %}
+{{ '  get-lease-hostnames ' + p.get_lease_hostnames + ';' }}
```
Comment (k0ste) [inline templates/dhcpd_pool.j2]: Add a few more whitespaces to avoid breaking of format.

**Review by k0ste — CHANGES_REQUESTED** [review]

File: templates/dhcpd_pool.j2 (hunk 1 above)
Comment (ProFfeSsoRr) [reply templates/dhcpd_pool.j2]: Ok, done

**Review by k0ste — APPROVED** [review]

**Pull request merged**